build.gradle

```diff
@@ -18,6 +18,13 @@ buildscript {
     }
 }
 
+plugins {
```

Comment:
just use the apply plugin below, or move the apply plugins here. consistency is nice

Reply:
how